documentation/docs/guide/openshift/change_password.md

```diff
@@ -0,0 +1,22 @@
+# Changing Passwords
+
+## Postgres
+* Log in to [Silver Cluster](https://console.apps.silver.devops.gov.bc.ca/k8s/cluster/projects).
+* Select the project you would like to change the password(s)for ie:
+  * `<licenseplate>-test`
+  * `<licenseplate>-dev`
+  * `<licenseplate>-prod`
+* Using the lefthand nav menu, navigate to:
+    ```
+    Administrator
+      |
+      Workloads
+        |
+        Pods
+    ```
+* Select the `pmo-postgres-deployment-<unique-identifier>` pod.
+* Enter the terminal for the pod.
+* In the terminal, use the command:
+```sh
+psql -U <postgres-user> -d <database> -c "ALTER USER <user-to-change> WITH PASSWORD '<new-password>';"
```

Comment:
does this work without entering the existing password, also for someone new to this project, where would the find the existing database, and exiting user

Reply:
I'm not sure it makes sense to have that information in the public-facing repo...

Reply:
you don't have to say, where exactly you can say something like get database from the kubernetes config, and the secret for the kubernetes secret